python/paddle/distributed/fleet/base/distributed_strategy.py

```diff
@@ -634,6 +649,26 @@ def localsgd(self, flag):
 
     @property
     def localsgd_configs(self):
+        """
+        Set LocalSGD training configurations. LocalSGD has a configurable
+        setting that can be configured through a dict.
+
+        **Notes**:
+            **k_steps(int)**: The local steps for training before parameter
+                synchronization. Default 1. If strategy.auto is set True, the
+                local steps will be calculated automatically during training.
```

Comment:
I think currently k_steps is hard to decide when auto is set. Is there a heuristic that we can set the k_steps automatically.

Reply:
[Adaptive Communication Strategies to Achieve the Best Error-Runtime Trade-off in Local-Update SGD](https://arxiv.org/pdf/1810.08313.pdf)
we have shown the reference. I will move it here

Reply:
done